### PR TITLE
Fix network mgmt cr and add proper descriptions

### PR DESCRIPTION
**Pull request by fahlmant**
Fixing Role CRs to match what's in the mockup. 

https://marvelapp.com/d780h9h/screen/64100971

### CODE REVIEW

File: deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 kind: AWSFederatedRole
 metadata:
-  name: customer-admin
+  name: network-mgmt
   namespace: aws-account-operator
 spec:
-  roleDisplayName: customer-admin
-  roleDescription: This is the description of customer-admin
+  roleDisplayName: Network Management
+  roleDescription: User can view all AWS resources, and can edit network-related resoruces to enable things like VPN, VPC, peering, etc.
```
Comment (jharrington22) [inline deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml]: "User can view all AWS resources, and can edit VPC related resources to enable services such as IPsec VPN and VPC peering."

File: deploy/crds/aws_v1alpha1_awsfederatedrole_readonly_cr.yaml [hunk 2]
```diff
@@ -4,8 +4,8 @@ metadata:
   name: read-only
   namespace: aws-account-operator
 spec:
-  roleDisplayName: read-only
-  roleDescription: This is the description of read-only
+  roleDisplayName: Read Only
+  roleDescription: User may view all AWS resources for this cluster, but not modify or create resources.
```
Comment (jharrington22) [inline deploy/crds/aws_v1alpha1_awsfederatedrole_readonly_cr.yaml]: "User may view all AWS resources for this cluster, but not modify, create or delete resources".

File: deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml [hunk 3]
```diff
@@ -1,11 +1,11 @@
 apiVersion: aws.managed.openshift.io/v1alpha1
 kind: AWSFederatedRole
 metadata:
-  name: customer-admin
+  name: network-mgmt
   namespace: aws-account-operator
 spec:
-  roleDisplayName: customer-admin
-  roleDescription: This is the description of customer-admin
+  roleDisplayName: Network Management
```
Comment (zgalor) [inline deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml]: "Network management"

File: deploy/crds/aws_v1alpha1_awsfederatedrole_readonly_cr.yaml [hunk 4]
```diff
@@ -4,8 +4,8 @@ metadata:
   name: read-only
   namespace: aws-account-operator
 spec:
-  roleDisplayName: read-only
-  roleDescription: This is the description of read-only
+  roleDisplayName: Read Only
```
Comment (zgalor) [inline deploy/crds/aws_v1alpha1_awsfederatedrole_readonly_cr.yaml]: "Read only"

File: deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml (hunk 3 above)
Comment (fahlmant) [reply deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml]: Is it a UI requirement for these to be quoted?

Comment (zgalor) [reply deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml]: No No.
I meant `Network management`.
see https://marvelapp.com/d780h9h/screen/64100971

Comment (fahlmant) [reply deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml]: Oh I see now. Will fix

**Comment by zgalor** [comment]
LGTM

**Pull request merged**